ext/random/randomizer.c

```diff
@@ -80,7 +85,7 @@ PHP_METHOD(Random_Randomizer, __construct)
 
 	ZVAL_OBJ(&zengine_object, engine_object);
 
-	zend_update_property(random_ce_Random_Randomizer, Z_OBJ_P(ZEND_THIS), "engine", strlen("engine"), &zengine_object);
+	zend_update_property(random_ce_Random_Randomizer, Z_OBJ_P(ZEND_THIS), "engine", sizeof("engine") - 1, &zengine_object);
```

Comment:
Not sure how this change is related, but that's still `strlen("engine")` in the <code>zend_<b>read</b>_property</code> call near the end of this file
https://github.com/php/php-src/blob/e9a447f9b30b05f462252a5527eaab46b6e9963e/ext/random/randomizer.c#L282

Reply:
Thank you. This is a digression from the purpose of this PR, but it was minor and has been corrected.
4afc9db8162b58051b271e2dd292aff09a6c3747

Reply:
Please don't bundle unrelated changes into a PR, it distracts from the relevant parts. Can you please revert the `strlen` / `sizeof` changes?

I also disagree with using `sizeof` in principle: Modern compilers can optimize both. For older compilers the string is very short and it's also not in a hot path. `strlen()` is much more readable, because the `- 1` is not needed.

Reply:
That's right.
But, the current situation where it is not unified is not so good, so we will create a separate PR to unify it with strlen.